### DISCUSSION TITLE
I have already Auto1111 installed. How do i update to your fork without having to clone repo and start from scratch ?

### DISCUSSION THREAD

**Discussion by ZZZephyriusss** [anapnoe/stable-diffusion-webui-ux#99] [General]
I am a newbie so sorry if my question sound ignorant.

 I have just come across your version and as a heavy Sd user, i will love to test your webui. But i have a very big SD install with tone of lora, models etc.. Do i have to start from the bottom with repo clon then import all the models or can i just fork with a command like

 git fetch origin pull/  "fork number /head:batch
git checkout batch

?

thanks if someone have time to respond

**Comment by Zuzakiin** [comment]
Don't do what I did originally and install ux directly over 1111
https://github.com/anapnoe/stable-diffusion-webui-ux/issues/30


another alternative to symlinks is using
 https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings
It looks like many folders can be specified in bat file
yeah, deleted rambling about symlinks and complications, this is the way to do it.

**Reply by ZZZephyriusss** [reply]
yeah, i specify a new path in the bat file for .ckp and embeddings etc.. and works fine ! Unfortunaly, UX doens't accept for now the extension OpenOutPaint tht i use primaly. So i will wait for the merge or an update. Hope Ux will be soon the new face of A1111 ! So much cleaner !!

**Comment by halr9000** [comment]
I run a bunch of them side by side, and I use a combo of symlinks and model_dir cmdline flags. But as noted above the extensions don't necessarily work, so I quit keeping those separate. My webui-user.bat for both auto and ux looks like this now:

`set COMMANDLINE_ARGS=--listen --enable-insecure-extension-access --no-half --api --xformers --update-all-extensions --ckpt-dir="D:\hal\stable-diffusion\models\Stable-diffusion" --lora-dir="D:\hal\stable-diffusion\models\Lora" --embeddings-dir="D:\hal\stable-diffusion\models\Embeddings" --hypernetwork-dir="D:\hal\stable-diffusion\models\Hypernetworks"`

So if I were starting with a new fork, what I would do is:

1. git clone
2. copy over webui-user.bat
3. run it once to setup venv etc (these can be shared as well, but auto updating dependencies is exactly what is breaking the forks, so I would not recommend it)
4. and then for extensions, either copy over the whole folder and restart to see what breaks, or
5. reinstall extensions one-by-one, testing each time

**Comment by drdancm** [comment]
You might want to try Stability Matrix which is made for running multiple Packages and share models. It works really well.
I have A111, and this new UI-UX, and Foocus, ComfyUI etc installed.
